src/backend/backend.did

```diff
@@ -78,6 +78,7 @@ type AddUserCredentialRequest = record {
 	current_user_version : opt nat64;
 	credential_spec : CredentialSpec
 };
+type AddUserCredentialResult = variant { Ok; Err : AddUserCredentialError };
```

Comment:
Reading through the PR, I am surprised that this is classified as a breaking change in candid.  If I get a moment I'll have a poke.  I'd need to see the exact change report.